src/TodoList.jsx

```diff
@@ -1,33 +1,14 @@
-import TodoListItem from './TodoListItem'
+import TodoListItem from './TodoListItem';
 
-const TodoList = () => {
-  const todoList = [{
-    id: 1,
-    title: "451 Fahrenheit",
-    author: "Ray Bradbury",
-    state: "California",
-    country: "USA"
-  }, {
-    id: 2,
-    title: "The Catcher in the Rye",
-    author: "Jerome David Sallinger",
-    state: "New Hampshire",
-    country: "USA"
-  }, {
-    id: 3,
-    title: "The Lord of Flies",
-    author: "William Golding",
-    state: "Cornwall",
-    country: "UK"
-  }];
+const TodoList = ({todoList}) => {
 
   return (
     <ul>
-      {todoList.map(item => {
-        return <TodoListItem todo={item} key={item.id} />
+      {todoList.map(({id, ...item}) => {
+        return <TodoListItem key={id} {...item} />
```

Comment:
Let's not spread item here (and on the map). It's easier to read if you assign title to its own prop.

Remember that readability is important. Code is read and tweaked more than it's written. Readable code also cuts down the learning curve for newbies and makes it easy for others to get what's going on and make changes.

Reply:
> Let's not spread item here (and on the map). It's easier to read if you assign title to its own prop.

Got it, thank you, redestructured it.